server/app/services/scheduler/error.rb

```diff
@@ -1,5 +1,3 @@
 module Scheduler
-  class Error < StandardError
-
-  end
+  Error = Class.new(StandardError)
```

Comment:
I guess I'm trying to say is: do we need to change these `class Error < ...` into something weirder and more unexpected?